Common/services/HTTPService.swift

```diff
@@ -59,11 +58,11 @@ class HTTPService {
 
         let task = session.dataTask(with: request, completionHandler: {(data, response, error) in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/cafeb7b8636cbafd0216eaa2489bb108aa52959c/Common/services/HTTPService.swift#L59"></span>Unused parameter “response” in a closure should be replaced with _.<br />
<code>unused_closure_parameter</code> <code>HTTPService.swift:59</code></td>
    </tr>
  </tbody>
</table>